### PR TITLE
Add reacnetgenerator

### PR DESCRIPTION
**Pull request by njzjz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/reacnetgenerator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  license_family: LGPL
+  license_file: LICENSE
+  summary: 'An automatic generator of reaction network for reactive molecular dynamics simulation.'
+  doc_url: https://reacnetgenerator.njzjz.win/
+  dev_url: https://github.com/tongzhugroup/reacnetgenerator
+
+extra:
+  recipe-maintainers:
+    - njzjz
+    - DiracMD
```
Comment (njzjz) [inline recipes/reacnetgenerator/meta.yaml]: @DiracMD please confirme you are willing to be listed here.

Comment (DiracMD) [reply recipes/reacnetgenerator/meta.yaml]: yes?is it ok ?

**Comment by njzjz** [comment]
@conda-forge-admin, please restart ci

**Comment by njzjz** [comment]
@conda-forge-admin, please restart ci

**Comment by njzjz** [comment]
@conda-forge/help-python All CIs turn green. Please review the recipe. Thanks!

**Comment by njzjz** [comment]
@conda-forge/help-python-c ready to review

**Comment by npavlovikj** [comment]
Thank you for contributing @njzjz, the recipe LGTM!

**Pull request merged**